src/test/groovy/org/prebid/server/functional/tests/BidderParamsSpec.groovy

```diff
@@ -371,4 +375,149 @@ class BidderParamsSpec extends BaseSpec {
         and: "targeting should be empty"
         assert response.targeting.isEmpty()
     }
+
+    @PendingFeature
+    @Unroll
+    def "PBS should emit error when validate-media-types = #description and site-media-types is empty in bidder config"() {
+        given: "Pbs config"
+        def pbsService = pbsServiceFactory.getService(pbsConfig)
+
+        and: "Default basic BidRequest with zoneId = null"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            site = Site.defaultSite
+        }
+
+        when: "PBS processes auction request"
+        def response = pbsService.sendAuctionRequest(bidRequest)
+
+        then: "Response should contain empty seatbid"
+        assert response.seatbid.isEmpty()
+
+        and: "Response should contain error"
+        assert response.ext?.errors[ErrorType.GENERIC]*.code == [2]
+        assert response.ext?.errors[ErrorType.GENERIC]*.message ==
+                ["Site for bidder is not supported, request will be skipped"]
+
+        where:
+        pbsConfig                                               | description
+        ["adapters.generic.meta-info.validate-media-types": "true",
```

Comment:
This looks like it should be 2 separate tests as in one case you're setting validate-media-types and in other - not. Also try to not parametrize description.